synthtool/languages/node_mono_repo.py

```diff
@@ -59,6 +59,44 @@ def read_metadata(relative_dir: str):
         return data
 
 
+def copy_sample_to_quickstart(relative_dir):
```

Comment:
nit: maybe `copy_list_sample_to_quickstart`. Without the `list` in the name, one might assume this is a general utility to copy a sample

Reply:
Also, same comment on type-hints